domain/host/utils.go

```diff
@@ -133,7 +140,10 @@ func getIPResources(hostID string, hostIP string, staticIPs ...string) ([]HostIP
 	if err != nil {
 		return nil, err
 	}
-	glog.V(3).Infof("Interfaces on this host %s: %+v", hostID, ifacemap)
+	plog.WithFields(log.Fields{
+		"hostID": hostID,
+		"ifacemap": ifacemap,
```

Comment:
Where you're going to be logging the same fields over and over, redefine your logger at the top. For example:
```
plog := log.WithFields(logrus.Fields{
    "host": hostID,
})
...
log.WithFields(logrus.Fields{
    "interfaces": ifacemap,
}).Debug("Found host interfaces")